game.js

```diff
@@ -380,11 +387,22 @@ function main() {
   });
 
   const game = new Game({
-    width: 25,
-    height: 25,
-    squareSize: 20,
+    width: WIDTH,
+    height: HEIGHT,
+    squareSize: SQUARE_SIZE,
     players: [player1, player2],
   });
 
-  game.start();
+  const table = document.getElementById("table-stats");
+  table.width = WIDTH*SQUARE_SIZE;
```

Comment:
I'm not sure why we are constructing a table here and giving it properties of the game. If table is a specific component of the ui, then it should be constructed within the ui. If the table needs to be constructed based on game configs then it should ask the game for those configs